src/pages/sa-trx/[functionId]/forms/SaleTransactionForm.js

```diff
@@ -384,6 +400,13 @@ export default function SaleTransactionForm({ labels, access, recordId, function
           const basePriceValue = postMetalToFinancials === false ? basePrice : 0
           const TotPricePerG = basePriceValue
 
+          const unitPrice =
+            ItemConvertPrice?.priceType === 3
+              ? weight * TotPricePerG
+              : parseFloat(ItemConvertPrice?.unitPrice || 0).toFixed(3)
+
+          const minPrice = parseFloat(ItemConvertPrice?.minPrice || 0).toFixed(3)
+          formik.setFieldValue(`items[${newRow.id - 1}].minPrice`, minPrice)
```

Comment:
what the purpose? ypu're already setting minPrice using update, pls remove and test again